server.js

```diff
@@ -1,34 +1,85 @@
+/* eslint-disable linebreak-style */
 import express from 'express'
-import bodyParser from 'body-parser'
 import cors from 'cors'
+import listEndpoints from 'express-list-endpoints'
+
+import exercisesData from './data/exercises.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
-// Start defining your routes here
+// Lists all endpoints
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send(listEndpoints(app))
+})
+
+// Lists all exercises
+app.get('/exercises', (req, res) => {
+  res.json({ data: exercisesData })
+})
+
+// Endpoint to get a exercise by id
+app.get('/exercises/id/:id', (req, res) => {
```

Comment:
if you filtering by the primary key (like id) you don't need to specify it in the endpoint. `/exercises/:id` would be the best here